Source/WebKit/NetworkProcess/NetworkConnectionToWebProcess.messages.in

```diff
@@ -131,8 +131,8 @@ messages -> NetworkConnectionToWebProcess WantsDispatchMessage {
     NavigatorGetPushPermissionState(URL scopeURL) -> (Expected<uint8_t, WebCore::ExceptionData> result)
 #endif
 
-    [EnabledIf='isWebTransportEnabled()'] InitializeWebTransportSession(URL url) -> (std::optional<WebKit::WebTransportSessionIdentifier> identifier)
-    [EnabledIf='isWebTransportEnabled()'] DestroyWebTransportSession(WebKit::WebTransportSessionIdentifier identifier)
+    [Validator=isWebTransportEnabled] InitializeWebTransportSession(URL url) -> (std::optional<WebKit::WebTransportSessionIdentifier> identifier)
+    [Validator=isWebTransportEnabled] DestroyWebTransportSession(WebKit::WebTransportSessionIdentifier identifier)
```

Comment:
You should leave this as-is, since you are changing it in another PR to EnabledBy. Enabled by makes sense here.

Reply:
Yes, I am just changing them here so this patch could build and run test; I am going to drop these now that the other patch is merged. Thanks for review!